website/templates/project/project.mako

```diff
@@ -33,7 +33,7 @@
                             <a class="btn btn-default" data-bind="click: makePublic">Make Public</a>
                         % endif
                     % else:
-                        % if 'admin' in user['permissions']:
+                        % if 'admin' in user['permissions'] and not node['is_registration']:
```

Comment:
@HarryRybacki Yes, making registrations private should no longer be allowed. Can you update the `Node#set_privacy` method and the corresponding view to to raise a proper exceptions when this is attempted?